### PR TITLE
Add offline support for PWA

### PR DESCRIPTION
**Pull request by zecakeh**
This was the last step needed so the web app is fully considered a Progressive Web App.

I had to export some web config to have all features checked in Lighthouse. That's because Expo Web doesn't support ["maskable" icons](https://web.dev/maskable-icon/) yet.

Somehow Lighthouse says that our app takes 30 seconds before being interactive on a phone, which by experience isn't true. However I'm sure it could be optimized.

I managed to install it with Chromium and Brave on Linux.

Capture d’écran du 2021-01-17 16-33-58

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const createExpoWebpackConfigAsync = require("@expo/webpack-config");
+
+module.exports = async function (env, argv) {
+  console.log("env", env);
```
Comment (tasn) [inline webpack.config.js:4]: Is this a leftover debug print?

Comment (zecakeh) [reply webpack.config.js:4]: Oops, yes 😅

**Comment by tasn** [comment]
You resolved the conversation but haven't pushed the changes. :)

**Comment by zecakeh** [comment]
I did push it a while ago but VSCodium is still working on it… I'm gonna do it manually.

**Pull request merged**